scripts/deploy-backend.sh

```diff
@@ -89,9 +89,9 @@ update_backend()
 
         # Do a zero downtime deploy.  This requires enough memory for
         # two apps to exist in the org/space at one time.
-        cf push "$CGAPPNAME_BACKEND" --no-route -f manifest.buildpack.yml  --strategy rolling || exit 1
+        cf push "$CGAPPNAME_BACKEND" --no-route -f manifest.buildpack.yml -t 180 --strategy rolling || exit 1
     else
-        cf push "$CGAPPNAME_BACKEND" --no-route -f manifest.buildpack.yml
+        cf push "$CGAPPNAME_BACKEND" --no-route -f manifest.buildpack.yml -t 180
```

Comment:
Since these might be insufficient for prod and we aren't running migrate automatically, perhaps we should ultimately remove the increase of timeout from default (60) to 180?